src/DaPigGuy/PiggyCustomEnchants/enchants/tools/ExplosiveEnchant.php

```diff
@@ -12,7 +12,6 @@
 use pocketmine\inventory\Inventory;
 use pocketmine\item\Item;
 use pocketmine\Player;
-use ReflectionException;
```

Comment:
This class should ``use ReactiveTrait`` instead of extending ReactiveEnchantment
> Couldn't comment line 20, so I commented this line